Procfile

```diff
@@ -1 +1,3 @@
-bundle exec thin start -R config.ru -e $RACK_ENV -p $PORT
\ No newline at end of file
+deploy: bundle exec rake seed:admins
```

Comment:
I don't think so because if you look at the Oveview tab on the dashboard it shows web and bundle. The deploy command shown here isn't even being recognized by Heroku. This was something I was testing a while back. I thought I had removed it but I guess I didn't.